client/src/components/Notices/NoticeDetail.js

```diff
@@ -54,7 +63,25 @@ const _NoticeDetail = ({notice}: NoticeDetailProps) => {
     },
     {
       label: 'Objednávateľ',
-      body: notice.name,
+      body: (
+        <a className="notice-item-link" onClick={toggleCustomer}>
+          {showCustomerInfo ? <span>[&minus;]</span> : '[+]'} {notice.name}
+        </a>
+      ),
+    },
+    {
+      label: null,
+      body: showCustomerInfo ? (
+        <CompanyDetails eid={notice.eid} useNewApi />
+      ) : null,
+    },
+    {
+      label: 'Stav',
+      body: (
+        <span className={notice.total_final_value_amount ? 'text-danger' : 'text-success'}>
+          {notice.total_final_value_amount ? 'UKONČENÉ' : 'PREBIEHA'}
```

Comment:
Do we still want this, since there is another PR which removes finished notices from the list? Or will we keep it there for later? Let me know your opinion, I can't decide.

Reply:
At least, I wouldn't show `UKONČENÉ`, since all notices that can be open are finished.

Reply:
Oh, thanks a lot, @lenhardt. So keep it here then :slightly_smiling_face: